app/controllers/friends/base.js

```diff
@@ -11,8 +11,8 @@ export default Ember.Controller.extend({
   ),
 
   actions: {
-    save: function() {
-      if(this.get('isValid')) {
+    save() {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).